src/rowbinary/tests.rs

```diff
@@ -160,3 +176,22 @@ fn it_deserializes() {
         assert_eq!(actual, sample());
     }
 }
+
+#[test]
```

Comment:
I'd also add a separate time/64 it for a round-trip, including corner case values